news/2020-06-24-new-featured-artist-rusty-k.md

```diff
@@ -0,0 +1,63 @@
+---
+layout: post
+title: "New Featured Artist: Rusty K"
+date: 2020-06-24 08:00:00 +0000
+---
+
+We're proud to welcome **Rusty K** aboard as our newest Featured Artist!
+
+![](https://assets.ppy.sh/artists/87/header.jpg)
+
+Spinning electronic music marvels since 2012, [**Rusty K**](https://osu.ppy.sh/beatmaps/artists/87) is a Russian producer most well known in osu! from [Mir](https://osu.ppy.sh/users/8688812) and [C00L](https://osu.ppy.sh/users/4930630)'s vanguard maps set to *All That We Lost*, *Dark Eyes* and *Last Of Us* back in 2017 focusing on the exploration of a then-emerging subgenre of 'technical' mapping.
+
+We're super excited to unveil a selection of **14** tracks, including the timeless classics listed above. We're hoping that this new collection inspires more mappers to pick up the torch and continue experimenting with style and sound.
+
+Don't just take our word for it though, give these previews a listen and see for yourself:
+
+### All That We Lost
+
+Check out [this classic 2017 set](https://osu.ppy.sh/beatmapsets/655248) from [Mir](https://osu.ppy.sh/users/8688812)!
+
+<audio controls>
+    <source src="https://assets.ppy.sh/artists/86/previews/2230.mp3" type="audio/mpeg">
+</audio>
+
+### Dark Eyes
+
+Check out [this classic 2017 set](https://osu.ppy.sh/beatmapsets/652397) from [Mir](https://osu.ppy.sh/users/8688812)!
+
+<audio controls>
+    <source src="https://assets.ppy.sh/artists/86/previews/2230.mp3" type="audio/mpeg">
+</audio>
+
+### Last Of Us
+
+Check out [this classic 2017 set](https://osu.ppy.sh/beatmapsets/669153) from [C00L](https://osu.ppy.sh/users/4930630)!
+
+<audio controls>
+    <source src="https://assets.ppy.sh/artists/86/previews/2230.mp3" type="audio/mpeg">
+</audio>
+
+### Limitless
+
+<audio controls>
+    <source src="https://assets.ppy.sh/artists/87/previews/2342.mp3" type="audio/mpeg">
+</audio>
+
+### Frozen Heart
+
+<audio controls>
+    <source src="https://assets.ppy.sh/artists/87/previews/2340.mp3" type="audio/mpeg">
+</audio>
+
+---
+
+You all should know the drill by now - all of the above plus a whole bunch more are available this very moment over at [**Rusty K's Featured Artist listing**](https://osu.ppy.sh/beatmaps/artists/87), all pre-timed and just waiting for you to be cool like Mir and C00L and map them.
+
+Emboldened by your collective ribbing from last week's uh, 'not so challenging' hint, the fabled Hintern set a challenge most grand last Monday for this release with the ominous hint of merely 'feo19'. 
+
+Many guessed but only a few were truly correct, with tweeter @Lukaico taking top prize in deciphering that 'feo' referred to iron oxide (a type of *rust*) and 19 referred to the atomic number of potassium, which goes by the letter *K* on the periodic table. A few trailing guesses arrived on the mark afterwards from @REFORMEDawnsday, @AJT_main and @DeityPorg as well.
```

Comment:
Raja/Kaine (http://osu.ppy.sh/users/1786945, @karankaamaa on Twitter) got this clue right as well iirc. he's the first one to decipher this even if i'm not mistaken i suppose (it's just because he's privated his account his tweet is only visible to certain people)

![image](https://user-images.githubusercontent.com/36564236/85530709-58a73b80-b638-11ea-9dac-fbe10c987f2d.png)

Reply:
can't showcase private accounts when nobody can see the answer but a few people :(